hotel_reservation/models/hotel_reservation.py

```diff
@@ -3,6 +3,7 @@
 
 import time
 import datetime
+from datetime import timedelta
 from datetime import datetime as dtime
```

Comment:
@krishnaprajapati16894 multiple import statements, we can use one like
from datetime import datetime as dtime, timedelta